server/ip-tools.ts

```diff
@@ -512,14 +370,14 @@ export const IPTools = new class {
 				if (!hosts || !hosts[0]) {
 					if (ip.startsWith('50.')) {
 						resolve('comcast.net.res-nohost');
-					} else if (IPTools.checkPattern(rangeTelstra, ipNumber)) {
-						resolve('telstra.net.res-nohost');
+					} else if (ipNumber >= telstraRange.minIP && ipNumber <= telstraRange.maxIP) {
+						resolve(telstraRange.host);
```

Comment:
Is it really necessary to check the Telstra range separately (i.e. are there IPs on 101.160.0.0 - 101.191.255.255 that are not Telstra IPs and would be resolved by `dns.reverse()`)?

Reply:
I don't know what you mean?

Reply:
This is checked after all the other ranges.

Reply:
Oh, yes, I understand now. Yes, I do believe that's why it's done this way.

Reply:
OK, cool, I'll keep `telstraRange` as a const.